src/operators/UserOperations.ts

```diff
@@ -0,0 +1,48 @@
+import User, { UserInterface } from '../models/user';
+
+export const create = async (user: UserInterface) => {
```

Comment:
For all of the operations, I actually suggest not throwing/catching here, and just let the calling function catch and then send an error to the client.

Reply:
So by just catching and sending the error to the client, you mean like just catching the error and just doing like a `console.error` message?